t/mojo/daemon.t

```diff
@@ -161,6 +161,16 @@ $app->routes->any(
   }
 );
 
+my $whatever = 'Failed!';
+
+$app->hook(
+  before_worker_start => sub {
+    my ($app) = shift;
+    my $ok = ref $app && $app->isa('Mojolicious');
+    $whatever = $ok ? 'Whatever!' : 'Wrong type';
+  }
+);
+
```

Comment:
Where is the test?

Reply:
Goog catch! :1st_place_medal:

Reply:
That is fixed now.  New commit takes care of Mojo::UserAgent::Server as well.